fennel/sources/sources.py

```diff
@@ -255,7 +241,6 @@ def get(name: str) -> Kafka:
             sasl_mechanism="",
```

Comment:
qq why is securiy_protocol set to PLAINTEXT? I see sasl_mechanism as another literal being set to empty, should it have a default value too ?

Reply:
As of now this method is being executed in test cases/documentation. That is the reason I haven't updated this logic. I believe specifying default values here doesn't change much given bootstrap servers are empty and there is no username and password.